src/features/common/Layout/Forms/InlineFormDisplayGroup.tsx

```diff
@@ -8,6 +8,7 @@ const InlineFormGroup = styled('div')({
   rowGap: 24,
   alignItems: 'flex-start',
   flexWrap: 'wrap',
+  width: '100%',
   '&.InlineFormGroup--other': {
```

Comment:
Is this actually needed? The `div` would take 100% width by default.

Reply:
This was added to properly style the buttons, removing of which leads to below:
<img width="451" alt="image" src="https://github.com/Plant-for-the-Planet-org/planet-webapp/assets/72646230/70beaa3b-ab99-4c5d-9754-30f6ef24a050">